TypeTreeDumper.Client/IpcInterface.cs

```diff
@@ -17,5 +17,10 @@ public void WriteLine(string value)
         {
             Console.WriteLine(value);
         }
+
+        public void WriteLine(string value, params object[] args)

```

Comment:
`object` should be avoided in the IPC interface, as it can allow invalid types to be passed to the function (which will lead to an exception). This should probably be changed to `params string[] args` and any arguments to it changed to use `.ToString`.

Reply:
I think I misunderstood how the the IPC interface worked. I think it might be better to keep the interface consistent with `string.Format`, in which case, an extension method might work better?
```csharp
public static class Extensions
{
    public static void WriteLine(this IpcInterface server, string value, params object[] args)
    {
        server.WriteLine(string.Format(value, args));
    }
}
```

Reply:
I didn't even think of that, an extension method would absolutely be a better idea, yes.